packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -250,13 +250,31 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   /// To force the [pixels] to a particular value without honoring the normal
   /// conventions for changing the scroll offset, consider [forcePixels]. (But
   /// see the discussion there for why that might still be a bad idea.)
+  ///
+  /// See also:
+  ///
+  ///  * The method [correctBy], which is a method of [ViewportOffset] be used
+  ///    by viewport render objects to also correct the offset during layout
+  ///    without notifying its listeners.
+  ///  * The method [jumpTo], for making changes to position while not in the
+  ///    middle of layout and applying the new position immediately.
+  ///  * The method [animateTo], which is like [jumpTo] but animating to the
+  ///    distination offset.
   void correctPixels(double value) {
     _pixels = value;
   }
 
+  // See also [correctPixels] which is used by the [ScrollPosition] itself to
+  // set the offset initially during construction or after
+  // [applyViewportDimension] or [applyContentDimensions] is called.
```

Comment:
///?

Reply:
Ah, I didn't realize dartdoc'ing overrides actually works. Done

Reply:
oh we should also include the docs from the superclass, i didn't realize it was doing that. (These docs will replace the superclass docs.)